core/src/main/java/resource/fld/core-base.fld

```diff
@@ -13850,4 +13850,35 @@ You can find a list of all available timezones in the Lucee administrator (Setti
     		<type>boolean</type>
     	</return>
     </function>
+    
+    <function>
+		<name>StringReduce</name>
+		<member-name>reduce</member-name>
+		<class>lucee.runtime.functions.string.StringReduce</class>
+		<description>Iterates over every element of the string and calls the closure to work on the elements of the string. This function will reduce the string to a single value and will return the value.</description>
+    	<argument>
+			<name>string</name>
+			<type>string</type>
+			<alias>inputString, string</alias>
```

Comment:
alias for "string" is not necessary, it is already the name
add "object" as alias, as structReduce has